changelog.d/1017.misc.rst

```diff
@@ -0,0 +1,2 @@
+Removed broken link to pgp mirror that made the travis build fail.
+Reported and fixed by @{ffe4} (gh pr #{1017})
```

Comment:
```suggestion
Reported and fixed by @{ffe4} (gh pr #1017)
```

Why the curly bracket quotes around the issue number?

Reply:
Ah, I just copy pasted from the `CONTRIBUTING.md` and by mistake assumed they were syntax for the changelog. Fixed it for my other PR.

Reply:
Ah. Sorry to force-push to your repo. I wanted to merge this ASAP, so I didn't wait for a response. Thanks for the quick response (and for all the great work you've been doing on `dateutil` recently)!

Reply:
No worries about the force-push. I intend to stick around for a bit, so I'm just glad I can get my builds to pass.